mongodbatlas/resource_mongodbatlas_cloud_backup_schedule.go

```diff
@@ -0,0 +1,403 @@
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"cluster_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"policies": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"policy_item": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Optional: true,
+										Computed: true,
+									},
+									"frequency_interval": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+									"frequency_type": {
+										Type:         schema.TypeString,
+										Required:     true,
+										ValidateFunc: validation.StringInSlice([]string{"hourly", "daily", "weekly", "monthly"}, false),
+									},
+									"retention_unit": {
+										Type:         schema.TypeString,
+										Required:     true,
+										ValidateFunc: validation.StringInSlice([]string{"days", "weeks", "months"}, false),
+									},
+									"retention_value": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			// Optionals
+			"reference_hour_of_day": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+					v := val.(int)
+					if v < 0 || v > 23 {
+						errs = append(errs, fmt.Errorf("%q value should be between 0 and 23, got: %d", key, v))
+					}
+					return
+				},
+			},
+			"reference_minute_of_hour": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+					v := val.(int)
+					if v < 0 || v > 59 {
+						errs = append(errs, fmt.Errorf("%q value should be between 0 and 59, got: %d", key, v))
+					}
+					return
+				},
+			},
+			"restore_window_days": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"update_snapshots": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			// Only computed
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"next_snapshot": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceMongoDBAtlasCloudBackupScheduleCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*matlas.Client)
+	projectID := d.Get("project_id").(string)
+	clusterName := d.Get("cluster_name").(string)
+
+	performUpdate := false
+
+	// Reading default configuration
```

Comment:
this is what we want to get rid of.   I'm going to ask to setup a meeting on this to try to ensure what is needed here is understood by all.